config/initializers/datadog.rb

```diff
@@ -0,0 +1,14 @@
+if ENV['DATADOG_ENABLED']
```

Comment:
just to be super explicit, this will make it so that if this variable is set to _any value_ the integration will be activated.

Reply:
Casting ENV vars to booleans is something I'd love to discover a really compact and sharable solution for. It keeps coming up.